### DISCUSSION TITLE
Manually/programmatically deactivate snippet session?

### DISCUSSION THREAD

**Discussion by creasty** [neoclide/coc.nvim#4604] [Q&A] [answered]
Deactivation of the snippet session is done automatically but sometimes I’d like to exit it rather manually without moving cursor at all.

cf. https://github.com/neoclide/coc.nvim/blob/3014125e4ab7d35e08bbcde946dca4812b84f105/doc/coc.txt#L1064

**Comment by chemzqm** [comment, marked as answer]
Use `:call CocAction('snippetCancel')`
